config/300-pipelineparams.yaml

```diff
@@ -4,12 +4,12 @@ metadata:
   creationTimestamp: null
   labels:
     controller-tools.k8s.io: "1.0"
-  name: pipelineparams.pipeline.knative.dev
+  name: pipelineparamses.pipeline.knative.dev
 spec:
   group: pipeline.knative.dev
   names:
     kind: PipelineParams
-    plural: pipelineparams
+    plural: pipelineparamses
```

Comment:
hmm this will be hard to remember,  a little non-intuitive will all existing plurals. 
`kubectl get pipelineparamses`

I don't have a recommendation, just pointing it out.

Reply:
> hmm this will be hard to remember, a little non-intuitive with all existing plurals.

It doesn't seem like we can fix it unfortunately without changing the name or forking code-generator https://github.com/kubernetes/code-generator/issues/53

> On the second note, since spec contains a map of list, how about we rename it to pipelineparam?

I feel like this isn't much better since it contains more than one parameter - if we use `PipelineParam` the name is inaccurate; if we use `PipelineParamses` it's weird but still it is the plural version of that word ¯\_(ツ)_/¯

I've created #121 to find a better name